pkg/schema/definition/comment.go

```diff
@@ -5,7 +5,7 @@ import (
 	"strings"
 
 	"github.com/pkg/errors"
-	blackfriday "github.com/russross/blackfriday/v2"
+	"github.com/russross/blackfriday/v2"
```

Comment:
ah didn't know golang detects this, cool I guess

Reply:
I noticed this few days back only, with the help of GoLand IDE :)